FreeRTOS/FreeRTOS-Kernel
# Define (U)BaseType_t to 64-bit types on Windows

Description
-----------

This ensures that BaseType_t and UBaseType_t are correctly defined to 64-bit types on 64-bit Windows.

Test Steps
-----------
Built windows demo locally.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/710


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
